homeassistant/components/freedompro/const.py

```diff
@@ -0,0 +1,10 @@
+"""Constants for the Freedompro integration."""
+from datetime import timedelta
+
+COORDINATOR = "coordinator"
+DOMAIN = "freedompro"
+FREEDOMPRO_CONFIG = "freedompro_config"
+FREEDOMPRO_PARALLEL_UPDATES = 1
+FREEDOMPRO_SCAN_INTERVAL = timedelta(minutes=5)
+FREEDOMPRO_URL = "https://api.freedompro.eu/api/freedompro/accessories"
```

Comment:
Unused + see another comment.
[The coordinator update_interval is set to 1 minute](https://github.com/home-assistant/core/pull/46332/files#diff-7378423dd22bcf9b7fcb78464b5b559baa0d90906080bbda791a04876354c829R85). 
```suggestion
```